docs/types/standard-linter-v2.md

```diff
@@ -67,3 +67,7 @@ array.
 
 **A**: Use `['*']` as the `grammarScopes` value, and it'll be triggered
 regardless of the scope of the current file.
+
+**Q**: How do I make linter use last messages for my provider?
+
+**A**: Return `null` from your provider, returning `false` or `undefined` will result in an error.
```

Comment:
I'd cut out the part about `false` or `undefined`, this page is just meant to describe the right way to do things, no point in mentioning the wrong way 😛.